wsgi.py

```diff
@@ -44,30 +51,15 @@ def get_openshift(self):
 
 
 def get_openshift():
-    url = os.environ["OPENSHIFT_URL"]
-    shift = MocOpenShiftSingleton(url, APP.logger).get_openshift()
+    shift = MocOpenShiftSingleton(OPENSHIFT_URL, APP.logger).get_openshift()
     return shift
 
 
 @AUTH.verify_password
 def verify_password(username, password):
-    """Validates a username and password.
-
-    WARNING: This function always succeeds (anybody can log in) if the auth
-    file is missing.
-    """
-
-    try:
-        with open("/app/auth/users", "r") as my_file:
-            user_str = my_file.read()
-        if user_str:
-            user = user_str.split(" ", 1)
-            if username == user[0] and password == user[1]:
-                return username
-    except FileNotFoundError:
-        return True
-
-    return False
+    """Validates a username and password."""
+
+    return username == ADMIN_USERNAME and password == ADMIN_PASSWORD
```

Comment:
Not sure as to you plan here.  I am assuming that you are creating secrets for the ADMIN_USERNAME and ADMIN_PASSWORD.

But what happens if the environment variables are changed while the service is running?  

I honestly don't like the use of globals, and don't know what your plan is to keep things secret on openshift instances.

Reply:
> Not sure as to you plan here. I am assuming that you are creating secrets for the ADMIN_USERNAME and ADMIN_PASSWORD.

I am assuming that in a Kubernetes deployment you have something like:

```
apiVersion: v1
kind: Secret
metadata:
  name: onboarding-credentials
  namespace: onboarding
type: Opaque
stringData:
  ACCT_MGR_ADMIN_USERNAME: admin
  ACCT_MGR_ADMIN_PASSWORD: pass
```

And that if you change it, you restart the service automatically using either `kustomize`, which gives you a new hash suffix on the secret, causing the deployment to restart the pod, or you're using something like [Reloader](https://github.com/stakater/Reloader) which accomplishes the same thing if the secret content changes.

> I honestly don't like the use of [environment variables]

That's the way cloud native applications are generally configured.